packages/react-vapor/src/utils/FilterUtils.spec.ts

```diff
@@ -45,7 +45,9 @@ describe('Utils', () => {
             });
 
             it('should return true if the item displayValue contain a exact match with the filterValue', () => {
-                expect(defaultMatchFilter(itemBoxWithDisplayValue.displayValue, itemBoxWithDisplayValue)).toBe(true);
+                expect(
+                    defaultMatchFilter(itemBoxWithDisplayValue.displayValue as string, itemBoxWithDisplayValue)

```

Comment:
Can you remove the cast here and change the `defaultMatchFilter` to accept a ReactNode?

Reply:
The first parameter of the defaultMatchFilter is the filter value, I don't see any use case where it could be a ReactNode